src/resmom_win/win/mom_mach.c

```diff
@@ -548,7 +548,7 @@ mem_sum(job *pjob)
 			nps = ji.TotalProcesses;
 		else
 			log_eventf(PBSEVENT_DEBUG3, PBS_EVENTCLASS_SERVER, LOG_DEBUG, __func__,
-				"QueryInformationJobObject for %d failed to get number of processes with error %lu", pjob->ji_hJob, GetLastError());
+				"QueryInformationJobObject for job %d, handle %d failed to get number of processes with error %lu", pjob->ji_qs.ji_jobid, pjob->ji_hJob, GetLastError());
```

Comment:
pjob->ji_qs.ji_jobid, it's a char buffer so please change your format specifier for it.